core/src/main/scala/chisel3/experimental/SerializableModuleGenerator.scala

```diff
@@ -17,6 +19,7 @@ trait SerializableModuleParameter
   * }}}
   * 3. user should guarantee the module is reproducible on their own.
   */
+@instantiable
```

Comment:
This changes the ABI of `SerializableModule`, but it should be safe enough, since it only provide the new feature.